README.md

```diff
@@ -97,7 +102,41 @@ Each lion should have:
 Create a new lion instance with the name `simba`
 
 ```ruby
-# code here
+class Animal
+  def initialize(name)
+    @name = name.capitalize
+  end
+
+  def greet
+    puts "Hello, my name is #{@name}"
+  end
+
+  def get_name()
+    @name
+  end
+
+  def set_name(name)
+    @name = name
+  end
+end
+
+class Lion < Animal
+  @@pack = []
+  attr_reader :king
+  def intitalize(name)
+      @name = name.capitalize
+      @@pack << self
+      if @name == "Simba"
+        @king = true
+      end
+  end
+end
+
```

Comment:
close, would finish that conditional up to include `@king = false` if @name != "Simba"